build error when 1st try the project

could you please check why I suffer this build error?

``` 
[root@localhost io-uring]# cargo run --package io-uring-test
    Updating crates.io index
  Downloaded remove_dir_all v0.5.3
  Downloaded tempfile v3.3.0
  Downloaded cfg-if v1.0.0
  Downloaded bitflags v1.3.2
  Downloaded fastrand v1.8.0
  Downloaded anyhow v1.0.68
  Downloaded once_cell v1.16.0
  Downloaded libc v0.2.139
  Downloaded socket2 v0.4.7
  Downloaded 9 crates (838.7 KB) in 1.12s
   Compiling libc v0.2.139
   Compiling anyhow v1.0.68
   Compiling io-uring v0.5.10 (/mnt/sda/sda5/wayne/rust/io_uring/io-uring)
   Compiling cfg-if v1.0.0
   Compiling bitflags v1.3.2
   Compiling remove_dir_all v0.5.3
   Compiling fastrand v1.8.0
   Compiling once_cell v1.16.0
   Compiling socket2 v0.4.7
   Compiling tempfile v3.3.0
   Compiling io-uring-test v0.1.0 (/mnt/sda/sda5/wayne/rust/io_uring/io-uring/io-uring-test)
error[E0658]: destructuring assignments are unstable
   --> io-uring-test/src/tests/register_buf_ring.rs:491:5
    |
491 |     _ = ring;
    |     ^
    |
    = note: see issue #71126 <https://github.com/rust-lang/rust/issues/71126> for more information

error[E0658]: destructuring assignments are unstable
   --> io-uring-test/src/tests/register_buf_ring.rs:491:7
    |
491 |     _ = ring;
    |     - ^
    |     |
    |     cannot assign to this expression
    |
    = note: see issue #71126 <https://github.com/rust-lang/rust/issues/71126> for more information

For more information about this error, try `rustc --explain E0658`.
error: could not compile `io-uring-test` due to 2 previous errors
[root@localhost io-uring]# 
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "io-uring"
-version = "0.5.10"
+version = "0.5.11"
 authors = ["quininer <quininer@live.com>"]
 edition = "2018"
 license = "MIT/Apache-2.0"

--- a/io-uring-test/src/tests/register_buf_ring.rs
+++ b/io-uring-test/src/tests/register_buf_ring.rs
@@ -488,7 +488,6 @@ where
     S: squeue::EntryMarker,
     C: cqueue::EntryMarker,
 {
-    _ = ring;
     // Create a BufRing
     // Register it
     // Unregister it

